src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -171,4 +192,46 @@ public void allowOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() thr
 
     assertThat(response.getJson().getString("action"), is("renewedThroughOverride"));
   }
+
+  @Test
+  public void forbidRenewalOverrideWhenFirstRequestIsNotRecall()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    DateTime loanDueDate = new DateTime(
+      2018, APRIL, 21,
+      11, 21, 43
+    );
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca, loanDueDate);
+
+    useLoanPolicyAsFallback(
+      loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId()
+    );
+
+    requestsClient.create(new RequestBuilder()
+      .hold()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(servicePointsFixture.cd1().getId())
+      .by(usersFixture.steve()));
+
+    requestsClient.create(new RequestBuilder()
+      .recall()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(servicePointsFixture.cd1().getId())
+      .by(usersFixture.charlotte()));
+
+    loansFixture.attemptOverride(
```

Comment:
There doesn't appear to be any checks in this test. How does this test identify that the override has been forbidden?

Reply:
There is one check for response status (inside #attemptOverride). And the error message is verified in other test class.

Reply:
What makes this test different to the other test?

If they are covering a different scenario, how do we know that the correct response is received?

Reply:
This test verifies that overriding is forbidden when request of another type is on the first place.
This should lead to "loan doesn't match any of override cases". 
I didn't add any check supposing that 422 status means "loan doesn't match any of override cases" error.
Now I'm thinking that it isn't the only case and there are other reasons for 422.
I'll add error message verification.

Reply:
Done